app/controllers/api/v2/courses_controller.rb

```diff
@@ -68,6 +64,14 @@ def update
 
     private
 
+      def first_draft_or_new_enrichment
+        if @course.enrichments.draft.any?
+          @course.enrichments.draft.first
+        else
+          @course.enrichments.new(status: 'draft')
```

Comment:
do you not need to clone the the last published enrichment?

Reply:
This is logic that already existed, I only moved it around because codeclimate is complaining